packages/o/openssl/makefile/patch.lua

```diff
@@ -0,0 +1,35 @@
+function _patch_for_llvm_rc(package, opt)
+    if package:is_plat("mingw") and package:has_tool("mrc", "llvm_rc", "llvm-rc", "rc") then
+        local cc = package:build_getenv("cc")
+        local cflags = opt.buildenvs.CFLAGS
+        local tmpfile = path.unix(os.tmpfile() .. ".c")
```

Comment:
we need to remove tmpfile after running this script

Reply:
OK